### DISCUSSION TITLE
112.0.5564.0 The daily construction has arrived

### DISCUSSION THREAD

**Discussion by whxuezhe** [Alex313031/Thorium-Win-AVX2#70] [General]
Since the 110 and 111 code libraries are not easy to translate the Thorium browser of AVX and AVX2 version, then study M112.0, which can be executed high. In addition, I do n’t just pay attention to the progress of the version; I actually want to know the development of the development of the development, the weekly code construction progress; Thanks!

**Comment by gz83** [comment]
@whxuezhe The last reminder: Do not repeatedly publish discussions or issues that urge updates. If related discussions or issues are found again, we will consider taking restrictive measures against you.

In addition, there is no problem of difficult compilation. If you need to pay attention to the progress, please check the commit in the repo
